.travis.yml

```diff
@@ -15,4 +15,11 @@
 language: python
 python:
   - "3.5"
-script: nosetests -w lib/
+install:
+  - pip install flake8
+  - pip install ansible-lint
+script:
+  - nosetests -w lib/
+  - ansible-lint -x 204,301
+  - flake8
+  - git ls-files | tr '\n' '\0' | xargs -0 -L1 bash -c 'if test "$(grep --files-with-matches --binary-files=without-match --max-count=1 --regexp='.*' "$0")" && test "$(tail --bytes=1 "$0")"; then echo "No new line at end of $0."; false; fi'
```

Comment:
Can we put this last line into a script in `src/scripts/cibuild` or similar?

Also, it might be better to put all these into a single script, and run them sequentially, because there's quite a bit of overhead in Travis CI to start separate tasks... compared to our expected run time for each of these.

Reply:
Done. Please let me know if you had further comments, or else please resolve the comment post which I can merge.

Reply:
Thanks for the update. I think the separate script is definitely better. I made a few additional notes, but will resolve this comment.